### ISSUE TITLE
Indentation guides not showing on empty lines (3.3.1)

### ISSUE THREAD

**Issue by alexanderweiss** [atg/chocolat-public#1793]
When enabling indentation guides, they will only show up on lines with actual tabs. So, when blank lines are trimmed, these lines don't show the indentation guides. The result is holes that make them quite a bit less helpful. I've added this quick graphical representation:

Expected:

```
class SomeClass {

    constructor() {
    |
    |   super()
    |
    |   return this
    |
    }

}
```

Actual:

```
class SomeClass {

    constructor() {

    |   hello()

    |   return this

    }

}
```

**Comment by ludwigschubert** [comment]
I always thought that was intentional to show there was no whitespace on these lines. But reading your issue I realize guides _would_ be more helpful if they worked like you suggest.

**Comment by atg** [comment]
Yes it wasn't intentional, just laziness on my part. Although secretly I quite like the way it works currently. It makes it easy to see which lines have what amount of whitespace. I guess if I fixed this it would have to be a preference.
